### DISCUSSION TITLE
StyleProps docs - hints

### DISCUSSION THREAD

**Discussion by snlacks** [mantinedev/mantine#6423] [Feature requests]
It'd be nice to have some hints in the editor/ide for style props.

Currently these appear like this:
![Screenshot 2024-06-20 092221](https://github.com/mantinedev/mantine/assets/6046561/9b33b567-ec85-44d3-833b-632f079f22a6)

By adding docs comments, we could get hints. 
![Screenshot 2024-06-20 092210](https://github.com/mantinedev/mantine/assets/6046561/e062b2dc-b1f5-4457-aa31-91909772ee19)

In the following example, the suggestions also include the doc text. I think this is where it'd be most useful.
![Screenshot 2024-06-20 092427](https://github.com/mantinedev/mantine/assets/6046561/bd4fa8c2-9968-4738-84d6-61a6405eba33)

I haven't applied all the props locally, just the ones to create the screen shots. I built the story book and the docs, there were no unintended consequences I can see. 

Can we add this? I can submit a PR. 

Downsides? This adds a a bunch of docs comments. It makes the hint popover bigger.

**Comment by rtivital** [comment]
You are welcome to submit a PR

**Reply by snlacks** [reply]
Thanks! Here's the PR: https://github.com/mantinedev/mantine/pull/6425
